components/Collections/src/MDCCollectionViewController.m

```diff
@@ -117,6 +117,23 @@ - (void)viewDidLoad {
                  withReuseIdentifier:classIdentifier];
 }
 
+- (void)viewDidLayoutSubviews {
+  [super viewDidLayoutSubviews];
+
+  // Fixes an iOS 11 bug where supplementary views would be given a zPosition of 1, meaning the
+  // scroll view indicator (with a zPosition of 0) would be placed behind supplementary views.
+  // We know that iOS keeps the scroll indicator as the top-most view in the hierarchy as a subview,
+  // so we grab it and give it a better zPosition ourselves.
+#if defined(__IPHONE_11_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0)
+  if (@available(iOS 11.0, *)) {
+    UIView *maybeScrollViewIndicator = self.collectionView.subviews.lastObject;
+    if ([maybeScrollViewIndicator isKindOfClass:[UIImageView class]]) {
```

Comment:
Is it not possible to give supplementary views a different zIndex? Like 2?

Reply:
This isn't a supplementary view - it's the scroll view indicator image view. iOS 11 appears to have introduced some logic that sets a zIndex on supplementary views automatically (likely to support tableview-style fixed headers).

Reply:
That's my concern - should we not lower the z-Index of our supplementary views instead of guessing that the last subview will be the scroll indicator?

Reply:
No because my understanding is that the supplementary view z index is being raised so that they appear above the cells to achieve the UITableView fixed headers effect.

Reply:
Ack.